lib/pronto/brakeman.rb

```diff
@@ -49,10 +50,27 @@ def severity_for_confidence(confidence_level)
       end
     end
 
-    def patch_for_warning(ruby_patches, warning)
-      ruby_patches.find do |patch|
+    def patch_for_warning(code_patches, warning)
+      code_patches.find do |patch|
         patch.new_file_full_path.to_s == warning.file.absolute
       end
     end
+
+    def run_all_checks?
+      pronto_brakeman_config['run_all_checks'] == true
```

Comment:
```suggestion
      pronto_brakeman_config['run_all_checks']
```
I think `== true` is redundant, unless you had something specific in mind?

Reply:
Just thought this would be nicer if it can either be true or false, rather than true or false or nil (if brakeman isn't included in the pronto config), but this works too -- I'll add a check just so this behaviour is documented.